packages/flutter_tools/templates/plugin/lib/projectName_method_channel.dart.tmpl

```diff
@@ -0,0 +1,19 @@
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+import 'package:meta/meta.dart';
```

Comment:
You shouldn't need `meta` here; `flutter/foundation.dart` vends common annotations.

(I'm not sure how you were able to compile this template, since `meta` isn't in the pubspec.)

Reply:
Not sure why it compiles, but meta is needed for @visibleForTesting.

Reply:
No it isn't; you can use `flutter/foundation.dart` as I said in my comment.

https://github.com/flutter/flutter/blob/master/packages/flutter/lib/foundation.dart#L20